pdal/pipeline.py

```diff
@@ -28,7 +28,10 @@ def __init__(
         spec: Union[None, str, Sequence[Stage]] = None,
         arrays: Sequence[np.ndarray] = (),
         loglevel: int = logging.ERROR,
+        json: str = None
```

Comment:
The strict annotation is `Optional[str] = None`

Reply:
An alternative would be to replace `None` with the empty string `""` as the default value.

Reply:
81203f1f1ae7ffed41bcdcaf057c4897f1a83b38